spec/sample_reports/blastn_sample/job.yaml

```diff
@@ -0,0 +1,8 @@
+--- !ruby/object:SequenceServer::BLAST::Job
+id: blastn_sample
+method: blastn
+
+databases:
+- !ruby/struct:SequenceServer::Database
+  name: "../database/sample/transcripts/Solenopsis_invicta/Sinvicta2-2-3.cdna.subset.fasta"
```

Comment:
I just took this from the asn file, in job it was absolute. In the asn it was like this except I had to replace spec with ..